specification/quota/resource-manager/Microsoft.Quota/preview/2023-06-01-preview/groupquota.json

```diff
@@ -0,0 +1,1823 @@
+            "$ref": "./examples/SubscriptionRequests/SubscriptionRequests_List.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/MgIdInParameters"
+          },
+          {
+            "$ref": "#/parameters/GroupQuotaNameInParameters"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success, with the details of the subscriptionId request.",
+            "schema": {
+              "$ref": "#/definitions/GroupQuotaSubscriptionIdList"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/managementGroups/{mgId}/providers/Microsoft.Quota/groupQuotas/{groupQuotaName}/subscriptionRequests/{requestId}": {
+      "get": {
+        "tags": [
+          "SubscriptionRequests"
+        ],
+        "summary": "Get the status of a single subscriptionId request by requestId.",
+        "description": "Get API to check the status of a subscriptionIds request by requestId.  Use the polling API - OperationsStatus URI specified in Azure-AsyncOperation header field, with retry-after duration in seconds to check the intermediate status. This API provides the finals status with the request details and status.",
+        "operationId": "SubscriptionRequests_Get",
+        "x-ms-examples": {
+          "SubscriptionRequests_Get": {
+            "$ref": "./examples/SubscriptionRequests/SubscriptionRequests_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/MgIdInParameters"
+          },
+          {
+            "$ref": "#/parameters/GroupQuotaNameInParameters"
+          },
+          {
+            "$ref": "#/parameters/RequestIdInParameters"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success, with the details of the subscriptionId request.",
+            "schema": {
+              "$ref": "#/definitions/GroupQuotaSubscriptionId"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/managementGroups/{mgId}/{resourceProviderScope}/providers/Microsoft.Quota/groupQuotas/{groupQuotaName}/groupQuotaLimits/{resourceName}": {
+      "put": {
```

Comment:
>put [](http://example.com/codeflow?start=7&length=3)

If your resource supports creation (PUT verb), it must support deletion (DELETE verb)

[https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html#common-issues-found-in-review](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html#common-issues-found-in-review)

Reply:
i'm not sure what a DELETE verb would look like for this endpoint seeing as how the resource sort of describes part of other resources/relationships between some resources?
@rahuls-microsoft would a DELETE perhaps be the same as just having 0 limit you think?

Reply:
The Quota reduction is not supported by RPS, so delete operation is not valid.

Reply:
Once the quota is increased, assigned to subscriptions ot GroupQuota, it cannot be deleted. Actually every subscription comes with default Quota, so It cannot be 0 in any case. The RPs don't support 0 value or quota reduction.